theme/src/components/text/_text.scss

```
@@ -1,2 +1,43 @@
-.cmp-text {}
-.cmp-text__paragraph {}
\ No newline at end of file
+.cmp-text {
+  margin: 0.75rem 0 0 0;
+  color: $text;
+  font-weight: normal;
+  font-size: 1em;
+  & b {
+    font-weight: bold;
+  }
+  & p a {
+    color: $accent;
+    text-decoration: none;
+    border: 0.125em solid $transparent;
+    border-radius: 0.25em;
+    &:hover {
+      color: $accent-hover;
+      text-decoration: underline;
+      text-decoration-thickness: 0.125em;
+      text-decoration-color: $accent-hover;
+      text-decoration-style: solid;
+    }
+    &:focus {
+      text-decoration: underline;
+      border-color: $accent;
+      outline: none;
+      text-decoration-thickness: 0.125em;
+      text-decoration-color: $accent;
+      text-decoration-style: solid;
+    }
+  }
```

```suggestion
  & p a {
    color: $color-link;
    text-decoration: none;
    border: 0.125em solid $transparent;
    border-radius: 0.25em;
    &:hover {
      color: darken($color-link, 20%);
      text-decoration: underline;
      text-decoration-thickness: 0.125em;
      text-decoration-color: darken($color-link, 20%);
      text-decoration-style: solid;
    }
    &:focus {
      text-decoration: underline;
      border-color: $accent;
      outline: none;
      text-decoration-thickness: 0.125em;
      text-decoration-color: $color-link;
      text-decoration-style: solid;
    }
  }
```

I think the `$color-link` variable is more appropriate? `$accent` seems like the Brand color, while `$color-link` is the color of anchor tags...